core/server/api/canary/redirects.js

```diff
@@ -23,7 +42,7 @@ module.exports = {
             cacheInvalidate: true
         },
         query(frame) {
-            return redirects.settings.setFromFilePath(frame.file.path)
+            return redirects.settings.setFromFilePath(frame.file.path, frame.file.ext)
```

Comment:
Don't think we need this parameter here. We are passing file path already, so inside of `setFromFilePath` method we can get the extension through `const {ext} = path.parse(filePath);`

Reply:
As for this, I'll check this out tomorrow. If I remember correctly, the `frame.file.path` doesn't have an extension in Windows.

Reply:
![image](https://user-images.githubusercontent.com/8130013/97933838-35935200-1db7-11eb-8598-6c93506972a0.png)

I printed out the `frame.file` and it became clear that we cannot use `frame.file.path` because there is no extension.

Reply:
Cool. Let's not change this for now than. Good to know where this method signature comes from :+1: